### ISSUE TITLE
Problems supplying port to script

### ISSUE THREAD

**Issue by robinandeer** [chrisantonellis/mongobar#1]
Hi, this seems like a really promising project!

I'm having some issues connecting to my database running on a non-standard port (27019).

When I add the details in the config:

```bash
$ mongobar config

┌───────────┬────────────────────────────┐
│ root      │ /var/lib/mongodbhc/backups │
│ log_level │ INFO                       │
└───────────┴────────────────────────────┘

Connections:

┌ [default] ┬──────────────────┐
│ host      │ localhost        │
│ port      │ 27019            │
│ username  │ admin            │
│ password  │ **************** │
│ authdb    │ admin            │
└───────────┴──────────────────┘
```

I get the following error:

```bash
[CRITICAL] Command failed: Command '['mongodump', '-h', 'localhost', '-p', '27019', '-u', 'admin', '-p', '*****************', '--authenticationDatabase', 'admin', '--db', 'scoutTest', '--out', '/var/lib/mongodbhc/backups/localhost:27019/goad-linseed', '--quiet', '--gzip']' returned non-zero exit status 1.
```

I notice that the `-p` option is showing up twice both for port and password which I assume is giving the error.

So I tried giving the port on the `mongobar` command line:

```bash
$ mongobar backup --port 27019 -d scoutTest
Traceback (most recent call last):
  File "/home/hiseq.clinical/miniconda/envs/prod170926/bin/mongobar", line 1255, in <module>
    MongobarScript().run()
  File "/home/hiseq.clinical/miniconda/envs/prod170926/bin/mongobar", line 121, in run
    getattr(self, self.args.action.replace("-", "_"))()
  File "/home/hiseq.clinical/miniconda/envs/prod170926/bin/mongobar", line 859, in backup
    metadata = self.mb.generate_metadata()
  File "/home/hiseq.clinical/miniconda/envs/prod170926/lib/python3.6/site-packages/mongobar/mongobar.py", line 77, in generate_metadata
    client = self.create_pymongo_client()
  File "/home/hiseq.clinical/miniconda/envs/prod170926/lib/python3.6/site-packages/mongobar/mongobar.py", line 64, in create_pymongo_client
    client = pymongo.MongoClient(**options)
  File "/home/hiseq.clinical/miniconda/envs/prod170926/lib/python3.6/site-packages/pymongo/mongo_client.py", line 413, in __init__
    raise TypeError("port must be an instance of int")
TypeError: port must be an instance of int
```

Probably just some small bugs - I will try to play around with the code a little and I can open a PR unless you fix it quicker than I 😉

**Comment by robinandeer** [comment]
Hmm, the config port problem only seems to be an issue with the latest pip release

However giving the port on the command line does raise the above exception from PyMongo

**Comment by chrisantonellis** [comment]
sorry for the delay, I'll get into this asap. You're correct, there are a few small bugs kicking around, thanks for bringing this to my attention

**Comment by chrisantonellis** [comment]
This issue should be fixed in version `0.0.12` available here in the master branch and on pip.
@robinandeer Please let me know at your convenience if this solves the problem so I can close the issue. Thanks!

**Comment by chrisantonellis** [comment]
I was able to duplicate and fix the issue. Thanks for the input 👍
